### PR TITLE
Add voucher design

### PR DESCRIPTION
**Pull request by talvasconcelos**
Allow to print LNURLw vouchers with a default LNbits design, or use a custom design of your own. The default, simple QR is still available. 

You can use the template.svg in `static/images/` folder to start a new custom design. 
voucher

### CODE REVIEW

**Comment by arcbtc** [comment]
Looking forward to merging and using once conflicts are resolved

File: lnbits/extensions/withdraw/crud.py [hunk 1]
```diff
@@ -26,9 +26,10 @@ async def create_withdraw_link(
             k1,
             open_time,
             usescsv,
-            webhook_url
+            webhook_url,
+            custom_url
         )
-        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
```
Comment (callebtc) [inline lnbits/extensions/withdraw/crud.py:32]: This doesn't need a migration?

File: lnbits/extensions/withdraw/static/js/index.js [hunk 2]
```diff
@@ -132,6 +137,9 @@ new Vue({
         id: this.formDialog.data.wallet
       })
       var data = _.omit(this.formDialog.data, 'wallet')
+      if (data.use_custom && !data?.custom_url) {
+        data.custom_url = '/static/images/default_voucher.png'
```
Comment (callebtc) [inline lnbits/extensions/withdraw/static/js/index.js:141]: Code style: maybe put the hard coded string into a constant (if that makes sense for JS).

File: lnbits/extensions/withdraw/static/js/index.js [hunk 3]
```diff
@@ -158,6 +165,10 @@ new Vue({
       data.title = 'vouchers'
       data.is_unique = true
 
+      if (data.use_custom && !data?.custom_url) {
+        data.custom_url = '/static/images/default_voucher.png'
```
Comment (callebtc) [inline lnbits/extensions/withdraw/static/js/index.js:169]: string should be const here?

**Pull request merged**